Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
```

Comment:
I wonder if this should go into some subdirectory -- otherwise, it could be misinterpreted to be necessary for building this project.

Reply:
Actually, should this be here at all? I see below that there _are_ some Dockerfiles in subdirectories...

Reply:
It shouldn't! Thanks!